inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -114,15 +141,22 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
     reg64_t reg_yoff = h_iter;
     reg64_t reg_xoff = r12;
 
+    Xbyak::Reg64 reg_load_table = r15;
+    Xbyak::Reg64 reg_load_store_mask = rcx;
```

Comment:
BTW, rcx is used as `param1` on Windows: https://github.com/openvinotoolkit/oneDNN/blob/a81b4753105bb0a1622790256b02f19916cce77c/src/cpu/x64/jit_generator.hpp#L100. It might be not a problem for your particular code, but might lead to a bug in case of future code modification, Recommendation here is to directly set `reg_load_store_mask = param1`

Reply:
Done